x/ccv/provider/keeper/throttle.go

```diff
@@ -270,7 +271,7 @@ func (k Keeper) SetThrottledPacketDataSize(ctx sdktypes.Context, consumerChainID
 
 	// Sanity check to ensure that the chain-specific throttled packet data queue does not grow too
 	// large for a single consumer chain. This check ensures that binaries would panic deterministically
-	// if the queue does grow too large.
+	// if the queue does grow too large. MaxThrottledPackets should be set accordingly (quite large).
 	if size >= uint64(k.GetMaxThrottledPackets(ctx)) {
 		panic(fmt.Sprintf("throttled packet data queue for chain %s is too large: %d", consumerChainID, size))
```

Comment:
Maybe we should use `Must` prefix for all functions which directly have a `panic`. So `SetThrottledPacketDataSize` becomes `Must..`

Reply:
I see where you're coming from here, and Marius also makes a good point below about implications of "must" for getters. Personally I could go either way. Maybe create an issue and we can address this at a broader level?